py-polars/polars/testing/asserts/series.py

```diff
@@ -13,6 +13,7 @@
 )
 from polars.datatypes.group import FLOAT_DTYPES
 from polars.exceptions import ComputeError, InvalidOperationError, ShapeError
+from polars.polars import assert_series_equal_py
```

Comment:
All imports from the polars.abi3 need to be put with `contextlib.suppress(ImportError)`. Like this:

https://github.com/pola-rs/polars/blob/39c86534522c963af391e0f0404b238f0d25048c/py-polars/polars/series/series.py#L113